cdap-docs/reference-manual/source/http-restful-api/query.rst

```diff
@@ -421,3 +442,87 @@ if results for the ``query-handle`` are attempted to be downloaded again.
      - The query handle does not match any current query
    * - ``409 Conflict``
      - The query results were already downloaded
+
+
+.. _http-restful-api-query-enable-disable:
+
+Enabling and Disabling Querying
+-------------------------------
+Querying (or exploring) of datasets and streams can be enabled and disabled using these endpoints.
+
+Exploration of data in CDAP is governed by a combination of enabling the CDAP Explore
+Service and then creating datasets and streams that are explorable. The CDAP Explore
+Service is enabled by a :ref:`setting in the CDAP configuration file
+<appendix-cdap-default-explore-service>` (``explore.enabled`` in ``cdap-site.xml`` file).
+
+Datasets and streams |---| that were created while the Explore Service was not enabled
+|---| can, once the service is enabled and CDAP restarted, be enabled for exploration by
+using these endpoints.
+
+You can also use these endpoints to disable exploration of a specific dataset or
+stream. The dataset or stream will still be accessible programmatically; it just won't
+respond to queries or be available for exploration using the CDAP UI.
+
+For datasets::
+
+  POST <base-url>/namespaces/<namespace>/data/explore/datasets/<dataset-name>/enable
+  POST <base-url>/namespaces/<namespace>/data/explore/datasets/<dataset-name>/disable
+
+For streams::
+
+  POST <base-url>/namespaces/<namespace>/data/explore/streams/<stream-name>/tables/<table-name>/enable
+  POST <base-url>/namespaces/<namespace>/data/explore/streams/<stream-name>/tables/<table-name>/disable
+
+Each of these endpoints returns a query handle that can be used |---| in the case of
+``enable`` |---| to :ref:`submit queries <http-restful-api-query-submitting>` and explore
```

Comment:
you don't use the handle to submit queries.  It is only used to track status of the query.